scripts/globals/magic.lua

```diff
@@ -340,19 +341,27 @@ function applyResistance(player,spell,target,diff,skill,bonus)
     if player:hasStatusEffect(EFFECT_ALTRUISM) and spell:getSpellGroup() == SPELLGROUP_WHITE then
         magicacc = magicacc + player:getStatusEffect(EFFECT_ALTRUISM):getPower();
     end
+	
     if player:hasStatusEffect(EFFECT_FOCALIZATION) and spell:getSpellGroup() == SPELLGROUP_BLACK then
         magicacc = magicacc + player:getStatusEffect(EFFECT_FOCALIZATION):getPower();
     end
     --difference in int/mnd
+	
     if (diff > 10) then
         magicacc = magicacc + 10 + (diff - 10)/2;
     else
         magicacc = magicacc + diff;
     end
-    --add acc for ele/dark seal
+	
+    --Add acc for dark seal
     if (player:getStatusEffect(EFFECT_DARK_SEAL) ~= nil and skill == DARK_MAGIC_SKILL) then
         magicaccbonus = magicaccbonus + 256;
     end
+	
+    --Add acc for klimaform
+    if (castersWeather == singleWeatherStrong[element] or castersWeather == doubleWeatherStrong[element] and player:getStatusEffect(EFFECT_KLIMAFORM) ~= nil) then

```

Comment:
should change that line to 
`if (player:getStatusEffect(EFFECT_KLIMAFORM) and (castersWeather == singleWeatherStrong[element] or castersWeather == doubleWeatherStrong[element])) then` 
to make the intention clearer (still all one line but github might wrap my post here). The ~= nil isn't needed, if they don't have the effect it will already evaluate "false" and if they due it'll be true so that lets you shorten the line a bit.